src/microbe_stage/MicrobeStage.tscn

```diff
@@ -15,11 +18,31 @@
 [ext_resource type="PackedScene" uid="uid://b5yuo8sj21lnf" path="res://src/engine/GuidanceLine.tscn" id="75"]
 [ext_resource type="PackedScene" uid="uid://dwkek50fju0fu" path="res://src/microbe_stage/PatchNameOverlay.tscn" id="85"]
 
-[node name="MicrobeStage" type="Node" node_paths=PackedStringArray("heatViewOverlay")]
+[sub_resource type="FastNoiseLite" id="FastNoiseLite_q12wg"]
+frequency = 0.2401
+
+[sub_resource type="NoiseTexture2D" id="NoiseTexture2D_d28l8"]
+noise = SubResource("FastNoiseLite_q12wg")
+
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_3c4mw"]
```

Comment:
This has seemingly appeared again so could you again reset the shader on the heat gradient plane to default?